src/OneDrive.Sdk.Authentication.WinStore/OnlineIdAuthenticationProvider.cs

```diff
@@ -22,61 +22,67 @@
 
 namespace Microsoft.OneDrive.Sdk
 {
+    using Microsoft.Graph;
+    using Microsoft.OneDrive.Sdk.Authentication;
+
     using System;
     using System.Collections.Generic;
     using System.Linq;
     using System.Threading.Tasks;
     using Windows.Security.Authentication.OnlineId;
 
-    public class OnlineIdAuthenticationProvider : AuthenticationProvider
+    public class OnlineIdAuthenticationProvider : MsaAuthenticationProvider
     {
-        private OnlineIdAuthenticator authenticator;
+        private readonly int ticketExpirationTimeInMinutes = 60;
```

Comment:
>ticketExpirationTimeInMinutes [](start = 29, length = 29)

Can we get the expiration time off of the ticket itself to avoid hardcoding this value?